packages/hoprd/src/admin.ts

```diff
@@ -73,9 +73,15 @@ export class AdminServer {
 
   async setup() {
     let adminPath = path.resolve(__dirname, '../hopr-admin/')
-    if (!fs.existsSync(adminPath)) {
+    let nextPath = path.resolve(adminPath, '.next')
+    if (!fs.existsSync(nextPath)) {
       // In Docker
       adminPath = path.resolve(__dirname, './hopr-admin')
+      nextPath = path.resolve(adminPath, '.next')
+      if (!fs.existsSync(nextPath)) {
+        console.log('Failed to start Admin interface: could not find NextJS app')
+        process.exit(1)
+      }
```

Comment:
What about this:

```
let adminPath
for(const adminRelPath of [ '../hopr-admin', './hopr-admin' ]) {
   adminPath = path.resolve(__dirname, adminRelPath)
   const nextPath = path.resolve(adminPath, '.next')
   if (!fs.existsSync(nextPath)) {
      delete adminPath
      continue
    }
 
if (!adminPath) {
        console.log('Failed to start Admin interface: could not find NextJS app')
        process.exit(1)
}
```

Reply:
we don't want to touch the FS, not delete anything from within hoprd